runtime/worker.rs

```diff
@@ -94,7 +93,8 @@ pub struct WorkerOptions {
   /// If not provided runtime will error if code being
   /// executed tries to load modules.
   pub module_loader: Rc<dyn ModuleLoader>,
-  pub npm_resolver: Option<Rc<dyn NpmResolver>>,
+  pub node_fs: Option<Arc<dyn deno_node::NodeFs>>,
```

Comment:
I think we'll need to do something similar with the `FileSystem` trait in `ext/fs` so that I can pass in a custom file system to the worker.